lib/typescript/httpclient/endpoints/listTags.ts

```diff
@@ -0,0 +1,28 @@
+import {doFetchFromBackend} from '../api';
+import {Tag, ColorTag} from '../model';
+import {ListTagsResponsePayload} from '../payload';
+
+const colorMapper = (color: string): ColorTag => {
+  const colors = ['BLUE', 'RED', 'GREEN', 'PURPLE'];
```

Comment:
I'd move out of the method definition. In order to simplify the code further we can also do something like:

```json5
  const tagMapper = {
  'BLUE': 'tag-blue',
  'RED': 'tag-red'
  // etc...
  }
```

that way we could remove the function all together as you could `tagMapper[color.toLowerCase()] || 'tag-blue'`